src/subdomains/generic/kyc/enums/kyc.enum.ts

```diff
@@ -65,6 +65,7 @@ export enum KycStepStatus {
   CANCELED = 'Canceled',
   IGNORED = 'Ignored',
   OUTDATED = 'Outdated',
+  DEACTIVATED = 'Deactivated',
 }
```

Comment:
I think we can use the `Canceled` status instead. Canceled steps are not shown in UI.